kubernetes/travis/deploy.sh

```diff
@@ -0,0 +1,28 @@
+export DEPLOY_BRANCH=${DEPLOY_BRANCH:-development}
+
+export REPOSITORY="https://github.com/${TRAVIS_REPO_SLUG}.git"
+
+sudo rm -f /usr/bin/git-credential-gcloud.sh
+sudo rm -f /usr/bin/bq
+sudo rm -f /usr/bin/gsutil
+sudo rm -f /usr/bin/gcloud
+rm -rf node_modules
+
+curl https://sdk.cloud.google.com | bash;
+source ~/.bashrc
+gcloud components install kubectl
+
+gcloud config set compute/zone us-central1-b
+openssl aes-256-cbc -K $encrypted_324c873b2450_key -iv $encrypted_324c873b2450_iv -in ./kubernetes/travis/susi-telegrambot-e467bca1e540.json.enc -out susi-telegrambot-e467bca1e540.json -d
+mkdir -p lib
+gcloud auth activate-service-account --key-file susi-telegrambot-e467bca1e540.json
+export GOOGLE_APPLICATION_CREDENTIALS=$(pwd)/susi-telegrambot-e467bca1e540.json
+gcloud config set project susi-telegrambot
+gcloud container clusters get-credentials bots
+cd kubernetes/images/generator
+docker build --build-arg COMMIT_HASH=$TRAVIS_COMMIT --build-arg BRANCH=$DEPLOY_BRANCH --build-arg REPOSITORY=$REPOSITORY --no-cache -t aliayubkhan/susi_telegrambot:$TRAVIS_COMMIT .
+docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+docker tag aliayubkhan/susi_telegrambot:$TRAVIS_COMMIT aliayubkhan/susi_telegrambot:latest
```

Comment:
@AliAyub007 here, change this to fossasia docker account and push docker image to it. I am creating a new repository in docker fossasia organization and messaging you the credentials of susi docker account. Store those credentials as travis environment variables. Also, use tag as `latest-$DEPLOY_BRANCH`